File: Simple/SimpleCalculator.java
import java.util.*;

public class SimpleCalculator {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter 1st number: ");
        int number1 = scan.nextInt();
        System.out.print("Enter 2nd number: ");
        int number2 = scan.nextInt();
        System.out.print("Enter Method of operation: ");
        char operator = scan.next().charAt(0);
        switch (operator) {
            case '+':
                System.out.println(number1 + number2);
                break;
            case '-':
                System.out.println(number1 - number2);
                break;
            case '*':
                System.out.println(number1*number2);
                break;
            case '/':
                System.out.println(number1/number2);
                break;

            default:
                break;
        }
        scan.close();
    }
}

File: Simple/SimpleCalculator.cpp
#include <iostream>

int main(int argc, char const *argv[])
{
    int number1;
    double number2;
    char operator_;
    std::cout<<"Enter First Number: "<<std::endl;
    std::cin>>number1;
    std::cout<<"Enter Second Number: "<<std::endl;
    std::cin>>number2;
    std::cout<<"Enter Method of Operation: "<<std::endl;
    std::cin>>operator_;

    switch (operator_)
    {
    case '+':
        std::cout<<"Result: "<<number1+number2<<std::endl;
        break;
    case '-':
        std::cout<<"Result: "<<number1-number2<<std::endl;
        break;
    case '*':
        std::cout<<"Result: "<<number1*number2<<std::endl;
        break;
    case '/':
        std::cout<<"Result: "<<number1/number2<<std::endl;
        break;
    
    default:
        std::cout<<"Operator not exist!"<<std::endl;
        break;
    }
    


    return 0;
}
